templates/index.html

```
@@ -0,0 +1,89 @@
+        <h2>Add Note</h2>
+        <form method="POST">
+            <textarea name="note" placeholder="Write your note here..."></textarea>
+            <button type="submit">Save Note</button>
+        </form>
+    </div>
+
+    <!-- ASK QUESTION -->
+    <div class="section">
+        <h2>Ask Question</h2>
+        <form method="POST">
+            <input type="text" name="question" placeholder="Ask something about your notes...">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)